common/beerocks/bwl/nl80211/base_wlan_hal_nl80211.cpp

```diff
@@ -764,6 +764,7 @@ bool base_wlan_hal_nl80211::refresh_vaps_info(int id)
             VAPElement vap_element;
 
             // Try reading the BSSID and SSID of the requested VAP
```

Comment:
Please fix the comment.
Either remove it or change to something generic:
```suggestion
            // Read values of the requested VAP
```

Reply:
Fixed